install_deps.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/sh
 set -e
+# Check whether running Ubuntu >= 18.04
+if test `lsb_release -rs` = '18.04';
```

Comment:
Just a minor comment that when Ubuntu 18.10 and 19.04 etc. are released, most likely they will continue to work with this new value for LIBSSL, but this script will use the older package name for those.  Changing the 'if' condition to something more like this might be good:
```
if [[ `lsb_release -rs` > "18" ]]
```